packages/extension/src/newtab/ShortcutLinks.tsx

```diff
@@ -187,6 +206,88 @@ export default function ShortcutLinks({
     isShortcutsV1,
   ]);
 
+  const onOptionsOpen = () => {
+    setShowOptions(true);
+
+    trackEvent({
+      event_name: AnalyticsEvent.OpenShortcutConfig,
+      target_type: TargetType.Shortcuts,
+    });
```

Comment:
Not blocking, but nice to have test case for it :)

Reply:
It is the old code I just moved it up, but I'll see if I can add test yes 👌

Reply:
We have the test for this already